### PR TITLE
Add new thermostat temperature device

### PR DESCRIPTION
**Pull request by Pierre-Gilles**
### Pull Request check-list

To ensure your Pull Request can be accepted as fast as possible, make sure to review and check all of these items:

- [x] If your changes affects code, did your write the tests?
- [x] Are tests passing? (`npm test` on both front/server)
- [x] Is the linter passing? (`npm run eslint` on both front/server)
- [x] Did you run prettier? (`npm run prettier` on both front/server)
- [x] If you are adding a new features/services, did you run integration comparator? (`npm run compare-translations` on front)
- [x] Did you test this pull request in real life? With real devices? If this development is a big feature or a new service, we recommend that you provide a Docker image to [the community](https://community.gladysassistant.com/) for testing before merging.
- [x] If your changes modify the API (REST or Node.js), did you modify the API documentation? (Documentation is based on comments in code)
- [x] If you are adding a new features/services which needs explanation, did you modify the user documentation? See [the GitHub repo](https://github.com/GladysAssistant/v4-website) and the [website](https://gladysassistant.com).
- [x] Did you add fake requests data for the demo mode (`front/src/config/demo.js`) so that the demo website is working without a backend? (if needed) See [https://demo.gladysassistant.com](https://demo.gladysassistant.com).

NOTE: these things are not required to open a PR and can be done afterwards / while the PR is open.

### Description of change

Kapture 2022-10-14 at 13 29 54

### CODE REVIEW

File: front/src/components/boxs/device-in-room/device-features/ThermostatDeviceFeature.jsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
+import get from 'get-value';
+import { Text } from 'preact-i18n';
+
+import { getDeviceName } from './utils';
+import { DeviceFeatureCategoriesIcon } from '../../../../utils/consts';
+
+const NumberDeviceFeature = ({ children, ...props }) => {
```
Comment (atrovato) [inline front/src/components/boxs/device-in-room/device-features/ThermostatDeviceFeature.jsx]: Bad name according to filename.

An another option would be to merger with NumberDeviceFeature component, adding 2 options (step as number and displayUnit = true/false)

**Review by atrovato — CHANGES_REQUESTED** [review]

File: front/src/components/boxs/device-in-room/device-features/ThermostatDeviceFeature.jsx (hunk 1 above)
Comment (Pierre-Gilles) [reply front/src/components/boxs/device-in-room/device-features/ThermostatDeviceFeature.jsx]: I think we want to keep this as a separate component, as I changed the code to make it more specific :)

**Review by VonOx — CHANGES_REQUESTED** [review]
Component look fine, but I think we need to implement 2 features for thermostats.

* Temperature ( Actual measured température ) `DEVICE_FEATURE_TYPES.THERMOSTAT.TEMPERATURE`
* Target Temperature `DEVICE_FEATURE_TYPES.THERMOSTAT.TARGETTEMPERATURE` 

Display both on the same components is a must have but can be implemented later.

So I suggest renaming to TARGETTEMPERATURE

**Comment by Pierre-Gilles** [comment]
@VonOx Thanks for the review! Indeed, TARGET_TEMPERATURE is more clear

**Comment by Pierre-Gilles** [comment]
@VonOx I updated with the requested changes

**Pull request merged**